.github/workflows/python.yml

```diff
@@ -0,0 +1,23 @@
+name: Pylint
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  pylint:
+    name: Run Pylint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - name: Install dependencies
+        run: pip install -r requirements.prod.txt
+      - name: Run pylint
+        run: pylint --rcfile=.pylintrc crawler backend
```

Comment:
Würde das in `ci.yml` integrieren
Und pylint 2 mal aufrufen. Einmal für crawler und einmal für bacaken dass man direkt weiß wo der fehler liegt